python/ssm_patch_compliance.py

```diff
@@ -0,0 +1,142 @@
+#
+# This file made available under CC0 1.0 Universal (https://creativecommons.org/publicdomain/zero/1.0/legalcode)
+#
+# Run SSM patch compliance check against ec2 instances
+# Description: Checks ec2 instances are patch compliant via SSM against SSM patch baseline.
+#
+# Trigger Type: Periodic
+# Scope of Changes: N/A
+# Required Parameter name: PoliciesToCheck
+# Required Parameter value example: Name bastion (Tag key and value for EC2 instances). It will find all instances with that particular
+# tag and for each instance it will insert a compliant/non compliant evaluation into AWS config.
+# IAM role running this lambda should be able to call ec2:describe , ssm:list_compliance_items and config:putEvaluations
+
+import json
+import boto3
+
+
+
+
+APPLICABLE_RESOURCES = ["AWS::EC2::Instance"]
+
+
+def validate_patch_baseline(instanceid):
```

Comment:
Split this function in two which return a boolean: 1) checking the presence of an agent, 2) checking the presence of an unpatched instance.